package.json

```diff
@@ -51,20 +51,25 @@
     "lodash": "^4.17.21",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.2",
-    "release-it": "^17.0.0",
+    "release-it": "^18.0.0",
     "sinon": "^9.0.1",
     "tmp": "^0.1.0",
     "vitest": "^0.33.0"
   },
   "peerDependencies": {
-    "release-it": "^14.0.0 || ^15.2.0 || ^16.0.0 || ^17.0.0"
+    "release-it": "^14.0.0 || ^15.2.0 || ^16.0.0 || ^17.0.0 || ^18.0.0"
+  },
+  "overrides": {
+    "@release-it-plugins/lerna-changelog": {
+      "release-it": "$release-it"
+    }
   },
   "engines": {
-    "node": ">= 16"
+    "node": ">= 20"
```

Comment:
This seems fine to me, but would you mind removing the dropped versions from the actions matrix too?